fastlane/.env.SAMPLE

```diff
@@ -0,0 +1,14 @@
+# Set if pushing to remote other than origin
+#FL_GIT_PUSH_REMOTE=
+
+# Needed for creating pull request
+GITHUB_PULL_REQUEST_REPO=RevenueCat/purchases-android
+GITHUB_PULL_REQUEST_API_TOKEN=
+
+# Optional for creating pull request
+GITHUB_PULL_REQUEST_ASSIGNEES=
+GITHUB_PULL_REQUEST_REVIEWERS=
+GITHUB_PULL_REQUEST_TEAM_REVIEWERS=
+
+# Needed for access to changelog and other git operations
+GITHUB_TOKEN=
```

Comment:
Honestly, we could probably use the same token here and the PR API token. I kept them separate since `GITHUB_PULL_REQUEST_API_TOKEN` is used by fastlane internally to create the PRs, while `GITHUB_TOKEN` is used on our side.

Reply:
maybe we can add a comment here to suggest that? i.e.: 
```bash
# Needed for access to changelog and other git operations
# It can be the same value as GITHUB_PULL_REQUEST_API_TOKEN
```